src/configuration/structures.ts

```diff
@@ -79,15 +82,9 @@ export enum BotStatus {
   Offline = "offline",
 }
 
-export interface GetBotResponse {
-  bot_agent: BotAgentDetails;
-}
-
-export interface BotAgentDetails extends BotAgent {
+export interface BotDetails extends Bot {
   default_group_priority: GroupPriority;
-  application: object;
-  max_chats_count: number;
-  groups: GroupConfig[];
+  client_id: string;
```

Comment:
i think it needs to be `owner_client_id`(this field is called like that in response from api), in go you could rename it with json tags but here it is not possible

Reply:
☝️